observability/otel/otel-configuring-otelcol-metrics.adoc

```diff
@@ -34,7 +45,7 @@ spec:
 
 You can use the *Administrator* view of the web console to verify successful configuration:
 
-* Go to *Observe* -> *Targets*, filter by *Source: User*, and check that the *ServiceMonitors* in the `opentelemetry-collector-<instance_name>` format have the *Up* status.
+* Go to *Observe* -> *Targets*, filter by *Source: User*, and check that the *ServiceMonitors* or *PodMonitors* in the `opentelemetry-collector-<instance_name>` format have the *Up* status.
```

Comment:
There is a few steps in this paragraph. Consider breaking them out into individual steps.

Reply:
Thank you, Darragh.
I'll add a suggestion to do just so.